tsl/src/bgw_policy/drop_chunks_api.c

```diff
@@ -105,10 +290,15 @@ drop_chunks_add_policy(PG_FUNCTION_ARGS)
 							get_rel_name(ht_oid))));
 		}
 
-		if (!DatumGetBool(DirectFunctionCall2(interval_eq,
-											  IntervalPGetDatum(&existing->fd.older_than),
-											  IntervalPGetDatum(older_than))) ||
-			(existing->fd.cascade != cascade) ||
+		same_interval =
```

Comment:
This should also happen in interval.c/.h and be able to check two interval types for equallity